app/views/plantings/index.html.haml

```diff
@@ -22,40 +22,71 @@
-        %td= link_to planting.crop.name, planting.crop
-        %td= link_to planting.garden.name, planting.garden
-        %td= planting.quantity
-        %td= planting.planted_at
+        %td= "##{index+1}"
+        %td= link_to image_tag((planting.default_photo ?  planting.default_photo.thumbnail_url : 'placeholder_150.png'), :alt => planting.crop_id, :class => 'img'), planting
+        %td
+          %ul{:style => "list-style-type:none"}
+            %li Owner :
+            %li Garden :
+            %li Planted on :
+            %li Finished on :
+            %li Sun/shade? :
+            %li Planted from :
+        %td
+          %ul{:style => "list-style-type:none"}
+            %li= link_to planting.owner.login_name, planting.owner
+            %li= link_to planting.garden.name, planting.garden
+            %li= planting.planted_at
+            - if planting.finished and planting.finished_at?
+              %li= planting.finished_at
+            - elsif planting.finished
+              %li= "Yes (no date specified)"
+            - else
+              %li= "(no date specified)"
+            - if planting.sunniness?
+              %li= planting.sunniness
+            - else
+              %li= "n/a"
+            %li= planting.planted_from
         %td
-          - if planting.finished and planting.finished_at
-            = planting.finished_at
-          - elsif planting.finished
-            Yes (no date specified)
-        %td= planting.sunniness
-        %td= planting.planted_from
+          %ul{:style => "list-style-type:none"}
+            %li= link_to 'Details', planting, :class => 'btn btn-default btn-xs'
+            - if can? :edit, planting
+              %li= link_to 'Edit', edit_planting_path(planting), :class => 'btn btn-default btn-xs'
+              - if ! planting.finished
+                %li= link_to "Mark as finished", planting_path(planting, :planting => {:finished => 1}),  :method => :put, :class => 'btn btn-default btn-xs append-date'
+            - if can? :destroy, planting
+              %li= link_to 'Delete', planting, method: :delete, data: { confirm: 'Are you sure?' }, :class => 'btn btn-default btn-xs'
+          %tr
         %td
-          = link_to 'Details', planting, :class => 'btn btn-default btn-xs'
-          - if can? :edit, planting
-            =link_to 'Edit', edit_planting_path(planting), :class => 'btn btn-default btn-xs'
-            - if ! planting.finished
-              = link_to "Mark as finished", planting_path(planting, :planting => {:finished => 1}),  :method => :put, :class => 'btn btn-default btn-xs append-date'
-          - if can? :destroy, planting
-            =link_to 'Delete', planting, method: :delete, data: { confirm: 'Are you sure?' }, :class => 'btn btn-default btn-xs'
+        %td
+          %ul{:style => "list-style-type:none"}
+            %li
+              Crop name:
+              =link_to planting.crop.name, planting.crop
+            - if planting.finished?
```

Comment:
There's a lot of moderately complex logic shared between here and your changes in  app/views/plantings/show.html.haml: it would be better if you could factor out the common logic into a helper function, or perhaps a method on the Planting model.